samples/XCT.Sample/App.xaml.cs

```diff
@@ -14,7 +15,7 @@ public App()
 			LocalizationResourceManager.Current.Init(AppResources.ResourceManager);
 
 			InitializeComponent();
-			MainPage = new BaseNavigationPage(new WelcomePage());
```

Comment:
I think it needs to be reverted

Reply:
@pictos ?

Reply:
@oscarjaergren, yes?

Reply:
Just wondering if we have considered the scenario if the user wants to draw in multiple places. 

I.e for example a signature control, in my case O Jargren. 
Wouldn't be possible to draw at the moment, as you would have to draw OJargren, as currently the control deletes all points when clicking a new drawing. 

Maybe we can add a bindable property to just say "OneLineDrawing = false/true"

Reply:
I think it can be added later. It will require to add additional properties: Strokes (List<List<Point>>) because if we need to restore points, we don't want line 1 be connected to line 2.

Reply:
Agree with @VladislavAntonyuk that can be an improvement after this gets merged.